api/tests/src/services/users/test_get_user_api_keys.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime, timedelta
+
+from src.adapters import db
+from src.services.users.get_user_api_keys import get_user_api_keys
+from tests.src.db.models.factories import UserApiKeyFactory, UserFactory
+
+
+def test_get_user_api_keys_empty_result(enable_factory_create, db_session: db.Session):
+    """Test get_user_api_keys returns empty list when user has no API keys."""
+    user = UserFactory.create()
+
+    api_keys = get_user_api_keys(db_session, user.user_id)
+
+    assert api_keys == []
+
+
+def test_get_user_api_keys_single_key(enable_factory_create, db_session: db.Session):
+    """Test get_user_api_keys returns single API key for user."""
+    user = UserFactory.create()
+    api_key = UserApiKeyFactory.create(user=user, key_name="Test Key")
+
+    api_keys = get_user_api_keys(db_session, user.user_id)
+
+    assert len(api_keys) == 1
+    assert api_keys[0].api_key_id == api_key.api_key_id
+    assert api_keys[0].key_name == "Test Key"
+    assert api_keys[0].user_id == user.user_id
+
+
+def test_get_user_api_keys_multiple_keys(enable_factory_create, db_session: db.Session):
+    """Test get_user_api_keys returns all API keys for user."""
+    user = UserFactory.create()
+
+    api_key1 = UserApiKeyFactory.create(
+        user=user, key_name="First Key", created_at=datetime.utcnow() - timedelta(hours=2)
+    )
+    api_key2 = UserApiKeyFactory.create(
+        user=user, key_name="Second Key", created_at=datetime.utcnow() - timedelta(hours=1)
+    )
+    api_key3 = UserApiKeyFactory.create(
+        user=user, key_name="Third Key", created_at=datetime.utcnow()
+    )
```

Comment:
datetime.utcnow() is deprecated. We have a datetime_util with a utcnow() function that uses the newer way of setting this.

Won't have any meaningful difference except using the deprecated method puts warnings in our test output: https://github.com/HHS/simpler-grants-gov/actions/runs/16806507662/job/47600224925#step:8:191


```suggestion
    api_key1 = UserApiKeyFactory.create(
        user=user, key_name="First Key", created_at=datetime_util.utcnow() - timedelta(hours=2)
    )
    api_key2 = UserApiKeyFactory.create(
        user=user, key_name="Second Key", created_at=datetime_util.utcnow() - timedelta(hours=1)
    )
    api_key3 = UserApiKeyFactory.create(
        user=user, key_name="Third Key", created_at=datetime_util.utcnow()
    )
```